px4_ros2_cpp/test/integration/global_navigation.cpp

```diff
@@ -0,0 +1,103 @@
+/****************************************************************************
+ * Copyright (c) 2023 PX4 Development Team.
+ * SPDX-License-Identifier: BSD-3-Clause
+ ****************************************************************************/
+
+/**
+ * Summary:
+ *  Tests that the GlobalPositionMeasurementInterface successfully publishes valid measurements to the expected ROS 2 topic.
+ */
+
+#include <gtest/gtest.h>
+#include <rclcpp/rclcpp.hpp>
+#include <px4_ros2/navigation/experimental/global_position_measurement_interface.hpp>
+#include "util.hpp"
+
+using namespace px4_ros2;
+
+class GlobalPositionMeasurementInterfaceTest : public Tester
+{
+protected:
+  void SetUp() override
+  {
+    node = initNode();
+
+    global_navigation_interface = std::make_shared<GlobalPositionMeasurementInterface>(*node);
+    ASSERT_TRUE(global_navigation_interface->doRegister()) <<
+      "Failed to register GlobalPositionMeasurementInterface.";
+
+    subscriber = node->create_subscription<px4_msgs::msg::VehicleGlobalPosition>(
+      "/fmu/in/aux_global_position", 10,
+      [this](const px4_msgs::msg::VehicleGlobalPosition::SharedPtr msg) {
+        received_measurement = msg;
+      });
+  }
+
+  void reset_measurement()
+  {
+    received_measurement = nullptr;
+  }
+
+  void sendGlobalPositionMeasurement(
+    std::unique_ptr<GlobalPositionMeasurement> measurement,
+    const std::string & test_subname = "")
+  {
+    ASSERT_NO_THROW(global_navigation_interface->update(*measurement)) <<
+      "Failed to send position measurement update via GlobalPositionMeasurementInterface.";
+
+    constexpr auto timeout_duration = std::chrono::milliseconds(200);
```

Comment:
If you place above `using namespace std::chrono_literals;`, you can do:
```suggestion
    const auto timeout_duration = 200_ms;
```

Reply:
Won't clang tidy complain about doing a namespace using-directives in this case?